### DISCUSSION TITLE
图片caption对齐方式

### DISCUSSION THREAD

**Discussion by alicelwy** [sjtug/SJTUThesis#923] [提问]
目前的模板中，图片的caption似乎是加粗版宋体（加粗版新罗马）且居中。
请问，如何将caption设置为“两端对齐”？以及，如何修改caption的字体格式（不想要加粗，想要楷体）

**Comment by note286** [comment]
`caption`包设置。

**Reply by alicelwy** [reply]
谢谢，我已经成功将caption的加粗去掉了，而且可以设置两端对齐了。但是我还有3个问题不知道怎么解决：
1. 中文图例如何改成楷体？我尝试在caption-font  .initial:n = \zihao{5} \normalfont这一句中添加\kaiti，但是没有成功；
2. 目前我使用的bicaption，所以中文图例和英文图例的所有属性相同，如何将英文的字体改为times new roman？
3. 如何实现中文图例居中但英文图例两端对齐？
希望您能为我提供更详细的解答，谢谢

**Reply by alicelwy** [reply]
第一个小问题，我刚刚解决了，是在\normalfont后面加上\kaishu

**Reply by cell-lives** [reply]
可以参考 #623 改回去

**Reply by ujust4fun** [reply]
> 谢谢，我已经成功将caption的加粗去掉了，而且可以设置两端对齐了。但是我还有3个问题不知道怎么解决：
> 
> 1. 中文图例如何改成楷体？我尝试在caption-font  .initial:n = \zihao{5} \normalfont这一句中添加\kaiti，但是没有成功；
> 2. 目前我使用的bicaption，所以中文图例和英文图例的所有属性相同，如何将英文的字体改为times new roman？
> 3. 如何实现中文图例居中但英文图例两端对齐？
>    希望您能为我提供更详细的解答，谢谢
@alicelwy 问题2和3解决了吗🥺？求教
